rtb-house/rtb-house-htb.js

```diff
@@ -438,11 +438,11 @@ function RTBHouseHtb(configs) {
             },
 
             /* The bid price unit (in cents) the endpoint returns, please refer to the readme for details */
-            bidUnitInCents: 1,
+            bidUnitInCents: 100,
```

Comment:
upewnię się - to znaczy że bidujemy w CPM?

Reply:
Oni tu chcą tak jakby 'rząd wielkości, żeby na koniec wyszło w centach'. "The unit is in terms of cents, thus it should be set to 100 if your endpoint returns dollars".
A jak rozumiem, bidujemy w dolarach. Potwierdź proszę i zresolvuj Ty, bo to na tyle duża kwestia, że wolę się upewnić.